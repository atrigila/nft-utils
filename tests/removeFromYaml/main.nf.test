nextflow_pipeline {

    name "Test removeFromYaml"
    script "./main.nf"
    tag "removeFromYaml"

    test("removeFromYaml") {
        when {
            params {
                outdir = "$outputDir"
            }
        }

        then {
            assert snapshot(removeFromYaml("$outputDir/pipeline_info/nf_core_pipeline_software_mqc_versions.yml", "Workflow", "Nextflow")).match()
        }
    }
}
