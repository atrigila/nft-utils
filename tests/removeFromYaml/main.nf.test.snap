{
    "removeFromYaml": {
        "content": [
            {
                "Workflow": {
                    "Pipeline": "1.0.0"
                }
            }
        ],
        "meta": {
            "nf-test": "0.9.0",
            "nextflow": "24.09.0"
        },
        "timestamp": "2024-10-07T17:00:34.208865"
    }
}